test/PushNotificationTests/BaseTestSuite.cpp

```diff
@@ -21,19 +21,31 @@ using namespace winrt::Microsoft::Windows::PushNotifications;
 
 void BaseTestSuite::ClassSetup()
 {
-    ::Test::Bootstrap::Setup();
+    ::Test::Bootstrap::SetupPackages();
+}
+
+void BaseTestSuite::ClassCleanup()
+{
+    ::Test::Bootstrap::CleanupPackages();
+}
+
+void BaseTestSuite::MethodSetup()
+{
+    ::Test::Bootstrap::SetupBootstrap();
```

Comment:
The same concern here as with the other PR. If the configuration is per class and not per method, I would expect a single process to be running all the tests. That would mean that this can be moved to classsetup.

Reply:
Right, but since each TEST_METHOD is being run as a separate process, it has to be done in MethodSetup.